pkg/sql/logictest/testdata/logic_test/builtin_function

```diff
@@ -1361,6 +1361,17 @@ SELECT DECODE('123\000456', 'escape')::STRING
 ----
 \x31323300343536
 
+query TT
+SELECT ENCODE('abc', 'base64'), DECODE('YWJj', 'base64')
```

Comment:
Can you include a test with decoding invalid base64?

Reply:
Also check what the behavior is with different amounts of padding.

Reply:
@justinj  Sure, should I force the error message to match PG? i.e. "ERROR:  invalid base64 end sequence
HINT:  Input data is missing padding, is truncated, or is otherwise corrupted."

Reply:
I don't have strong feelings, if the error from the library is reasonable I think it's safer and easier to just pass that through

Reply:
@justinj @knz added a decode that is invalid, one with padding1 and one with padding of 2